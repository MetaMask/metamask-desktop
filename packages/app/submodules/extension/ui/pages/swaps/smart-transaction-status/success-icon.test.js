import React from 'react';

import { renderWithProvider } from '../../../../test/jest';
import SuccessIcon from './success-icon';

describe('SuccessIcon', () => {
  it('renders the SuccessIcon component', () => {
    const { container } = renderWithProvider(<SuccessIcon />);
    expect(container).toMatchSnapshot();
  });
});
