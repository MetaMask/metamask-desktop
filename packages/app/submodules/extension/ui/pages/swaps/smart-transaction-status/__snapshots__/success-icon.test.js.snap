// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessIcon renders the SuccessIcon component 1`] = `
<div>
  <svg
    fill="none"
    height="39"
    viewBox="0 0 39 39"
    width="39"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M34.5457 19.8983C34.5457 23.9296 32.9443 27.7958 30.0937 30.6463C27.2432 33.4969 23.377 35.0983 19.3457 35.0983C15.3144 35.0983 11.4482 33.4969 8.59768 30.6463C5.74713 27.7958 4.1457 23.9296 4.1457 19.8983C4.1457 15.867 5.74713 12.0008 8.59768 9.15029C11.4482 6.29974 15.3144 4.69832 19.3457 4.69832C20.7897 4.69832 22.1957 4.90732 23.5257 5.28732L26.5087 2.30432C24.3047 1.39232 21.8917 0.898315 19.3457 0.898315C16.8506 0.898315 14.3799 1.38977 12.0747 2.3446C9.76953 3.29944 7.67499 4.69897 5.91067 6.46329C2.34748 10.0265 0.345703 14.8592 0.345703 19.8983C0.345703 24.9374 2.34748 29.7702 5.91067 33.3333C7.67499 35.0977 9.76953 36.4972 12.0747 37.452C14.3799 38.4069 16.8506 38.8983 19.3457 38.8983C24.3848 38.8983 29.2175 36.8965 32.7807 33.3333C36.3439 29.7702 38.3457 24.9374 38.3457 19.8983H34.5457ZM11.5747 16.2503L8.8957 18.9483L17.4457 27.4983L36.4457 8.49832L33.7667 5.80032L17.4457 22.1213L11.5747 16.2503Z"
      fill="#28A745"
    />
  </svg>
</div>
`;
